packages/example-app-base/src/components/Page/sections/ExamplesSection.tsx

```diff
@@ -27,7 +32,9 @@ export const ExamplesSection: React.StatelessComponent<IExamplesSectionProps> =
             const { view, ...exampleProps } = example;
             return (
               <div key={example.title + '-key'} className={styles.subSection}>
-                <ExampleCard {...exampleProps}>{view}</ExampleCard>
+                <ExampleCard {...exampleProps} onClick={handleClick} isCodeVisible={exampleProps.title === activeEditor}>
```

Comment:
>exampleProps.title [](start = 84, length = 18)

It's a little strange that we need to compare strings of all editors to figure out active editor. It should be some boolean flag inside of exampleProps, that specifies which editor is active.

Reply:
This check is effectively passing in a boolean isCodeVisible as a prop for each ExampleCard so it accomplishes the same goal. The check needs to be done here in ExamplesSection though to pass the property down to each ExampleCard.

---
In reply to: [302632279](https://github.com/OfficeDev/office-ui-fabric-react/pull/9769#discussion_r302632279) [](ancestors = 302632279)